src/CoreWCF.Kafka.Client/src/CoreWCF/ServiceModel/Channels/KafkaSecurityBindingElement.Config.cs

```diff
@@ -0,0 +1,375 @@
+    {
+        get => Config.SslCipherSuites;
+        set => Config.SslCipherSuites = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslCurvesList"/>
+    /// </summary>
+    public string SslCurvesList
+    {
+        get => Config.SslCurvesList;
+        set => Config.SslCurvesList = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslSigalgsList"/>
+    /// </summary>
+    public string SslSigalgsList
+    {
+        get => Config.SslSigalgsList;
+        set => Config.SslSigalgsList = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslCaCertificateStores"/>
+    /// </summary>
+    public string SslCaCertificateStores
+    {
+        get => Config.SslCaCertificateStores;
+        set => Config.SslCaCertificateStores = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslCrlLocation"/>
+    /// </summary>
+    public string SslCrlLocation
+    {
+        get => Config.SslCrlLocation;
+        set => Config.SslCrlLocation = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslProviders"/>
+    /// </summary>
+    public string SslProviders
+    {
+        get => Config.SslProviders;
+        set => Config.SslProviders = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslEngineLocation"/>
+    /// </summary>
+    public string SslEngineLocation
+    {
+        get => Config.SslEngineLocation;
+        set => Config.SslEngineLocation = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslEngineId"/>
+    /// </summary>
+    public string SslEngineId
+    {
+        get => Config.SslEngineId;
+        set => Config.SslEngineId = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.EnableSslCertificateVerification"/>
+    /// </summary>
+    public bool? EnableSslCertificateVerification
+    {
+        get => Config.EnableSslCertificateVerification;
+        set => Config.EnableSslCertificateVerification = value;
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SslEndpointIdentificationAlgorithm"/>
+    /// </summary>
+    /// <exception cref="ArgumentOutOfRangeException"></exception>
+    public SslEndpointIdentificationAlgorithm? SslEndpointIdentificationAlgorithm
+    {
+        get => Config.SslEndpointIdentificationAlgorithm;
+        set
+        {
+            if (value.HasValue && !SslEndpointIdentificationAlgorithmHelper.IsDefined(value.Value))
+            {
+                throw new ArgumentOutOfRangeException(nameof(value));
+            }
+
+            Config.SslEndpointIdentificationAlgorithm = value;
+        }
+    }
+
+    /// <summary>
+    /// <inheritdoc cref="ClientConfig.SaslKerberosServiceName"/>
+    /// </summary>
+    public string SaslKerberosServiceName
+    {
+        get => Config.SaslKerberosServiceName;
+        set => Config.SaslKerberosServiceName = value;
+    }
```

Comment:
We do support Kerberos through CustomBinding

Reply:
I couldn't find test case, so was raising...

Reply:
@birojnayak Indeed Kerberos is not tested at CoreWCF side as we are here just delegating to Confluent.Kafka.
The idea is :
- All Confluent properties settings are exposed through KafkaTransportBindingElement so a user can set them with a CustomBinding. In that case we do not have unit tests because these tests would be testing mappings. Basically this means all the features supported by Confluent.Kafka are supported by CoreWCF.Kafka.
- There are common security scenario (TLS + SASL authentication, mutual TLS..) for which we want to bring premium support by allowing configuration directly through KafkaBinding. For these scenario we do have integration tests.
This strategy should allow users that already know Kafka works to find what they need depending of their scenario.
For users who do not know how Kafka works are familiar with WCF security concepts, the KafkaBinding will take care to setup the correct security mechanism in Kafka world.
Adding @mconnew as i think the above approach should be shared.